cli/__main__.py

```diff
@@ -0,0 +1,3 @@
+from .cli import main

```

Comment:
Put this `cli` module in `mcs_benchmark_data.cli`. There should only be one top-level module, `mcs_benchmark_data`. So this file becomes `mcs_benchmark_data/cli/__main__.py`.